### PR TITLE
Publish mlte 0.2.6

### PR DESCRIPTION
**Pull request by turingcompl33t**
This PR publishes MLTE `0.2.6`.

### CODE REVIEW

**Review by krmaffey — APPROVED** [review]
Great work Kyle, LGTM!

**Review by krmaffey — APPROVED** [review]
Approve

**Review by krmaffey — DISMISSED** [review]
Approving again

**Review by krmaffey — DISMISSED** [review]
LGTM (3x the charm?)

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 `mlte` (pronounced "melt") is a framework and infrastructure for machine learning evaluation.
 
-![Version Badge](https://img.shields.io/badge/release-v0.2.4-e19b38)
+![Version Badge](https://img.shields.io/badge/release-v0.2.5-e19b38)
```
Comment (JennyNiemann) [inline README.md:7]: should be 6

**Review by JennyNiemann — APPROVED** [review]
The badge is still 5 but alas! Impressed you found this Kyle

**Review by JennyNiemann — APPROVED** [review]

**Review by JennyNiemann — APPROVED** [review]

**Review by JennyNiemann — APPROVED** [review]

**Review by JennyNiemann — DISMISSED** [review]

**Pull request closed without merging**